src/libvirtApi/rhsm.js

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
"rhsm.js" is an odd name. That means "Red Hat Subscription Manager", but it's not talking to that. "rhel-images.js"?